src/tools/dart2js.md

```diff
@@ -197,11 +197,11 @@ To debug in Chrome:
    [How to set breakpoints.](https://developers.google.com/web/tools/chrome-devtools/debug/breakpoints/add-breakpoints)
 4. Reload your app.
 
-### Internet Explorer {#dart2js-debugging-ie}
+### Edge {#dart2js-debugging-ie}
 
-To debug in Internet Explorer:
+To debug in Edge:
 
-1. Update to the latest version of Internet Explorer. (Source-map support
+1. Update to the latest version of Edge. (Source-map support
    was added to IE in April 2014).
 2. Load **Developer Tools** (**F12**). For more information, see
    [Using the F12 developer tools.](https://msdn.microsoft.com/library/ie/bg182326(v=vs.85))
```

Comment:
I think the URL needs updating. Maybe this one?

```suggestion
   [Using the F12 developer tools.](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide)
```

Reply:
Should we delete the 4th item? I don't see Ctrl+Shift+E or "exception" in that page.

Reply:
Hi, we can keep the 4th item. You can find the Ctrl+Shift+E on this page : https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide/debugger